### PR TITLE
Issue94 textcssfixes

### LINKED ISSUE #94: Add CSS/JS for text, Links handling
Add default classes for handling text, centering, links, others

### PR DESCRIPTION
**Pull request by klivingston**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #94 

Test URLs:
- Before: https://main--clarkcountynv--aemsites.aem.live/
- After: https://issue94-textcssfixes--clarkcountynv--aemsites.aem.live/

Added css for text handling

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -114,6 +113,47 @@ function buildAutoBlocks(main) {
...
+    if (a.href !== a.textContent) {
+      const up = a.parentElement;
+      const twoup = a.parentElement.parentElement;
+      if (!a.querySelector('img')) {
+        // TODO:  comment below will remove button links, should see if
+        // we do css for no-button instead "Link (plain links, no buttons)"
+
+        // if (up.childNodes.length === 1 && (up.tagName === 'P' || up.tagName === 'DIV')) {
+        //   a.className = 'button'; // default
+        //   up.classList.add('button-container');
```
Comment (meejain) [inline scripts/scripts.js:132]: Can we avoid tweaking the original code from aem.js ?

**Review by meejain — CHANGES_REQUESTED** [review]
@klivingston  - Thank you for the efforts. Can we avoid updating any code which is coming from aem.js .

File: scripts/scripts.js (hunk 1 above)
Comment (klivingston) [reply scripts/scripts.js]: Fixed by adding class and handler:
if (!a.closest('.no-buttons')) {

**Review by tmorris-adobe — APPROVED** [review]
looks good!

**Review by meejain — CHANGES_REQUESTED** [review]
Pl do not have any style sheet within scripts.

**Review by klivingston — COMMENTED** [review]
done

**Review by meejain — APPROVED** [review]

**Pull request merged**